plaidml/exec/exec.h

```diff
@@ -90,7 +90,7 @@ class Executable {
     for (size_t i = 0; i < raw_outputs.size(); i++) {
       raw_outputs[i] = outputs[i].as_ptr();
     }
-    ffi::call_void(              //
+    return ffi::call_double(     //
```

Comment:
`ffi::call_double` is unnecessary, just use `ffi::call<double>`.

Reply:
Done.